metabase/metabase
# add metabot animation

[Product doc](https://www.notion.so/metabase/Allow-EE-customers-to-override-our-fun-product-voice-7788c18b79d84bed828fde40d408e26d)

## Changes

Adds Metabot show/hide setting animation.

## How to verify

- Run Metabase EE
- Go to http://localhost:3000/admin/settings/whitelabel
- Change Metabot visibility setting and ensure it animates the change

<video src="https://user-images.githubusercontent.com/14301985/174678186-e5e3e198-8355-4009-a99b-b0d0b3e08aa5.mov" />
